specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/examples/GetInformationAboutAManagedDisk.json

```diff
@@ -20,6 +20,7 @@
             "publisher": "test_test_pmc2pc1",
             "product": "marketplace_vm_test"
           },
+          "supportsHibernation": true,
```

Comment:
should securityProfile be added here?

Reply:
@anshulsolanki21

Reply:
Raised PR for this change : https://github.com/meet1993shah/azure-rest-api-specs/pull/14